app/helpers/hyku_addons/api_works_helper.rb

```diff
@@ -8,16 +8,13 @@ def creator_from_json(creator_json)
 
       return if creator_hash.blank?
 
-      key = "creator_institutional_email"
-
       creator_hash.map do |creator|
-        if creator[key] && creator[key].blank?
-          creator.delete(key)
+        if creator["creator_institutional_email"]&.blank?
```

Comment:
This isn't the same as the code that was there before. The previous code checked that the index was set and then that the value was "blank". You dont need to use `&` because `blank?` responds to `nil` and any other type that you will be dealing with here.

Reply:
ok I think maybe we need to figure out a better way to write it to please rubocop and do what we want. Ill take a look on Monday